lesson-1-intro-to-k8s/README.md

```diff
@@ -0,0 +1,325 @@
+# Lesson 1
+
+## Introduction to Kubernetes (k8s)
+
+Welcome to the first lesson of the Red Hat Czech's OpenShift course for beginners.
+
+
+## What will you learn on the course?
+
+* Kubernetes and how to use it.
+* OpenShift.
+* Web console, `oc`, `kubectl`.
+* Basics of containerization.
+* The most common objects and their purpose.
+* Deploying applications into a cluster.
+* Operating your application.
+* Observability.
+* Advanced concepts.
+
+## Outside of our scope
+
+* Installation of a cluster.
+* Helping you with problems during the course on non-Red Hat operating systems.
+* Basically any troubleshooting will be hard to do virtually, sorry!
+* Serving coffee.
+
+
+## Before we start...
+
+* [Developer Sandbox - play with OpenShift whenever you want](https://developers.redhat.com/developer-sandbox/get-started).
+* [Become friends with the k8s docs](https://kubernetes.io/docs/home/) and [OpenShift docs](https://docs.openshift.com/).
+* Please ask questions, comment, speak up, sing: let's make this fun and interactive.
+* Materials: [github.com/cool-people/openshift-beginners-workshop](https://github.com/cool-people/openshift-beginners-workshop/)
```

Comment:
I wonder whether we want to move to any of the RH orgs or create our own?

Reply:
We definitely thought about it but... did not execute on it so far.

Dorka looked into it and got pointed to Diane that she could provide us with space.